### DISCUSSION TITLE
Outputs of multiple simulations on the same chart

### DISCUSSION THREAD

**Discussion by fredmcelwee** [gama-platform/gama#77] [Q&A] [answered]
I would like to run two simulations and have the output of both displayed on the same chart during the simulation run. The [`Manipulate simulations` section of the `Run Several Simulations` documentation](https://gama-platform.org/wiki/RunSeveralSimulations) says that this is possible, and displays an image of this being done for an Ant Foraging model. However, there is no code included, and I have not been able to recreate it myself despite my best efforts. 

I am also aware that there is a way to do this using a batch experiment where the outputs are charted together after all simulations are completed (e.g. [this related discussion gama-platform/gama#3259](https://github.com/gama-platform/gama/discussions/3259) ). However, I would like to keep the experiment type as GUI and have the chart update each cycle.

Does anyone have any pointers about how to go about doing this? 

Many thanks!

**Comment by benoitgaudou** [comment, marked as answer]
This can be done using the permanent statement.
(You can have a search in the model library to this key word to have
example).

Cheers


Le mer. 9 nov. 2022 à 13:28, fredmcelwee ***@***.***> a
écrit :
